### PR TITLE
NAS-108271 / 20.12 / Telegram Alert Service

### PR DESCRIPTION
**Pull request by dafo90**
Telegram Alert Service referring to UI issue NAS-108271 (tested in local with my other pull request https://github.com/freenas/freenas/pull/5984).

### CODE REVIEW

**Comment by erinix** [comment]
sorry to leave you hanging for a few days but I am now reviewing your PR, I tried to do a test alert with my bot and I got this message
screenshot-20201119-095756

File: src/app/pages/system/alertservice/alert-service/alert-service.component.ts [hunk 1]
```diff
@@ -597,6 +600,37 @@ export class AlertServiceComponent {
...
+            when: [{
+              name: "type",
+              value: 'Telegram',
+            }]
+          }],
+          required: true,
+          validation: [Validators.required],
+        },
+        {
+          type: 'input',
```
Comment (erinix) [inline src/app/pages/system/alertservice/alert-service/alert-service.component.ts:620]: I notice you mention that this field takes a space/comma separated list of ids, we have another component type, the 'chip' that is preferable for these

**Comment by dafo90** [comment]
I change the type from `input` to `chip` and I refactored the Telegram chat IDs payload generator based on the new component.

**Comment by erinix** [comment]
alright cool thanks I'll test again :)

**Review by erinix — APPROVED** [review]
lgtm, thanks for your contribution :)

**Pull request merged**